docs/pages/desktop-access/getting-started.mdx

```diff
@@ -444,13 +444,36 @@ the filepath to the `der_ca_file` configuration variable.
   order to resolve this.
 </Admonition>
 
+Install Teleport on the host where you will run the Teleport Deskop Service:
+
+(!docs/pages/includes/install-linux.mdx!)
+
 In order to enable Desktop Access in Teleport, add the following section in
-`teleport.yaml` on your Linux server. For a detailed description of these
+`/etc/teleport.yaml` on your Linux server. For a detailed description of these
 configuration fields, see the
 [configuration reference](./reference/configuration.mdx) page.
 
-<Details title="Desktop Service Configuration" min="8.0" scopeOnly={true} scope={["oss", "enterprise"]}>
+<Tabs>
+<TabItem scope={["oss","enterprise"]} label="Self-Hosted">
+
+For Teleport Self-Hosted, Windows Desktop Service can connect to the auth service with a
+join token. The Proxy will connect to the desktop listen address port.
+
+First, generate a join token with the following command:
+
+```code
+$ tctl tokens add --type=windowsdesktop
+$ tctl status
+```
+
+Copy the join token and `ca_pin` to a file on the instance where you will run Windows Desktop
+Service, and then use the following configuration:
+
 ```yaml
+teleport:
```

Comment:
If we're going to put a full config here, this should use a v3 config just like we do below.

Reply:
yes, did that.  I also added to turn off ssh, auth and proxy since we don't have a way of starting windows desktop role to have those off. Those would have come on automatically otherwise.